pulp_container/app/registry.py

```diff
@@ -163,7 +156,8 @@ async def dispatch_tag(tag, response_headers):
         try:
             artifact = tag.tagged_manifest._artifacts.get()
         except ObjectDoesNotExist:
-            raise ArtifactNotFound(tag.name)
+            ca = tag.tagged_manifest.contentartifact_set.all()[0]
```

Comment:
this change is actually redundant if we 'protect' the manifests from being reclaimed.

Reply:
I have decided to enable reclaim for manifests as well, it required to tweak the logic in schema_conversion.

Reply:
What if the manifest is broken and has not artifact nor remote artifact?

Reply:
actually handler handles this just fine - 404will be raised https://github.com/pulp/pulpcore/blob/master/pulpcore/content/handler.py#L681